src/iop/highlights.c

```diff
@@ -2621,8 +2622,9 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_widget_set_tooltip_text(g->scales, _("increase to correct larger clipped areas.\n"
                                            "large values bring huge performance penalties"));
 
-  GtkWidget *monochromes = dt_ui_label_new(_("not applicable"));
-  gtk_widget_set_tooltip_text(monochromes, _("no highlights reconstruction for monochrome images"));
+  GtkWidget *monochromes = gtk_label_new(_("not applicable"));
+  gtk_widget_set_halign(monochromes, GTK_ALIGN_CENTER);
```

Comment:
I'm not sure about this. I'm not against but if we go this way we probably want to do that for all other modules with the same kind of restrictions. So I'll prefer having this out of this PR and create a new PR for the alignment, but not that some mode have long restriction description like `raw chromatic aberrations`:

![image](https://user-images.githubusercontent.com/467069/223078372-4c8a81ae-1140-4877-bc09-f3dd22bf40b9.png)

At the end I'm not sure the center alignment will work fine.

Reply:
OK. If alignment needs discussion, I've removed the corresponding commit from PR. Now PR only corrects the inaccuracy of the tooltip text. But I've also added renames to the relevant elements in the source code to more accurately reflect what they're for.